pull

```diff
@@ -86,241 +52,147 @@ while [ "$1" != "" ]; do
-fi
-if [ $software = 'laravel' ];
-then
-  rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include='*/' --exclude="*" $serverAddressAndPath/storage/app/ storage/app/
+    # Extract fileadmin from docker container
+    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
+        cd $serverPath/ && mkdir -p tmp
+        docker cp main-application-1:/var/www/html/$webDirectory/fileadmin/ tmp/
+    "
+    catch "Could not extract fileadmin."
+
+    # Download fileadmin
+    rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --exclude="_processed_" --exclude="_temp_" --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include="*.youtube" --include='*/' --exclude="*" $serverAddressAndPath/tmp/fileadmin/ $webDirectory/fileadmin/
+    catch "Could not download fileadmin."
+    color green "Successfully downloaded fileadmin."
 fi
 if [ $software = 'statamic' ];
 then
-  rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include='*/' --exclude="*" $serverAddressAndPath/public/assets/ public/assets/
-fi
-if [ $software = 'shopware6' ];
-then
-  rsync -chavzP --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include="*.webp" --include='*/' --exclude="*" $serverAddressAndPath/public/media/ public/media/
-fi
-echo ''
+    # Extract assets from docker container
+    ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
+        cd $serverPath/ && mkdir -p tmp
+        docker cp main-application-1:/var/www/html/public/assets/ tmp/
+    "
+    catch "Could not extract assets."
+    # Download assets
+    rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats `if [ '$excludePath' != '' ]; then echo --exclude="$excludePath"; fi` --include="*.jpg" --include="*.JPG" --include="*.jpeg" --include="*.png" --include="*.PNG" --include="*.gif" --include="*.svg" --include='*/' --exclude="*" $serverAddressAndPath/tmp/assets/ public/assets/
```

Comment:
I see that those `include`/`exclude` patterns were there in the old version, too, but I’m wondering why one wouldn’t want to download all the files

Reply:
Some projects contain dozens of videos and other documents that are not necessary for development.